code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -627,6 +627,26 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/M, alien)
 /datum/reagent/medicine/bicaridine/overdose_crit_process(mob/living/M, alien)
 	M.apply_damages(1, 3, 2)
 
+/datum/reagent/medicine/meralyne
+	name = "Meralyne"
+	id = "meralyne"
+	description = "Meralyne is a concentrated form of bicardine and can be used to treat extensive blunt trauma."
+	color = "#E6666C"
+	overdose_threshold = REAGENTS_OVERDOSE
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
+	scannable = TRUE
+
+/datum/reagent/medicine/meralyne/on_mob_life(mob/living/M, alien)
+	M.heal_limb_damage(4*REM,0)
```

Comment:
Optional, but good whitespace is nice.
```suggestion
	M.heal_limb_damage(4 * REM, 0)
```

Reply:
Committed before I saw the "commit suggestion" tab. Thanks.